src/classes/CanvasRenderingContext2D.js

```diff
@@ -659,7 +641,14 @@ export default class CanvasRenderingContext2D {
   }
 
   get fillStyle() {
-    return this._fillStyleStack[this._stackIndex];
```

Comment:
We should simply be able to return the stack value here.

Reply:
Reasoning: This is faster, otherwise accessing `fillStyle()` will cause MooColor to compute the string with each property access.

Reply:
solved this with a `serializeColor` method

Reply:
`fillStyle()` will now either return the serialized color value, or the Gradient/Pattern object